care/facility/api/serializers/facility.py

```diff
@@ -152,6 +152,41 @@ def create(self, validated_data):
         return super().create(validated_data)
 
 
+class FacilityHubSerializer(serializers.ModelSerializer):
+    hub = FacilityBareMinimumSerializer(read_only=True)
+    spoke = FacilityBareMinimumSerializer(read_only=True)
+    hub_id = serializers.UUIDField(write_only=True)
```

Comment:
```suggestion
    id = serializers.UUIDField(source="external_id", read_only=True)
    hub = ExternalIdSerializerField(queryset=Facility.objects.all(), required=True)
    hub_object = FacilityBareMinimumSerializer(source="hub", read_only=True)
    spoke_object = FacilityBareMinimumSerializer(source="spoke", read_only=True)
```
we follow this pattern for uuids and related objects